modules/apps/transfer/keeper/relay_forwarding_test.go

```diff
@@ -1243,6 +1246,16 @@ func (suite *ForwardingTestSuite) TestMultihopForwardingErrorAcknowledgement() {
 	result, err = pathCtoD.EndpointA.AcknowledgePacketWithResult(packetFromCtoD, ack)
 	suite.Require().NoError(err)
 
+	// Ensure that chainC has an ack.
+	storedAck, found := suite.chainC.App.GetIBCKeeper().ChannelKeeper.GetPacketAcknowledgement(suite.chainC.GetContext(), pathBtoC.EndpointB.ChannelConfig.PortID, pathBtoC.EndpointB.ChannelID, packetFromCtoD.GetSequence())
+	suite.Require().True(found, "chainC does not have an ack")
+
+	// And that this ack is of the type we expect (Error due to ReceiveEnabled param being false)
+	initialErrorAck := channeltypes.NewErrorAcknowledgement(types.ErrReceiveDisabled)
+	forwardErrorAck := internaltypes.NewForwardErrorAcknowledgement(packetFromCtoD, initialErrorAck)
```

Comment:
Hmm... shouldn't this have been an acknowledgement for `packetFromBtoC`? (which is the forwarded packet, right?)

Reply:
Ah, yes, cool. This looks now correct to me. It's correct that `NewForwardErrorAcknowledgement` takes the `packetFromCtoD` because the error message includes the source port and channel of the sent packet.